Sources/VitalDevices/DeviceReading/Libre1Reader.swift

```diff
@@ -39,6 +39,8 @@ public class Libre1Reader {
     let samples = payload.1.map(LocalQuantitySample.init)
     let sensor = Libre1Sensor.init(sensor: payload.0)
     
+    postGlucose(.libreBLE, samples: samples)
```

Comment:
are samples guaranteed to be non empty here? 🤔

Reply:
No added a check now.